core/store/src/genesis.rs

```diff
@@ -0,0 +1,137 @@
+//! This file contains helper functions for accessing flat storage data in DB
```

Comment:
```suggestion
//! This file contains helper functions for creating genesis data in DB
```

This is not relevant to flat storage?..

Reply:
Not sure how this happened, I had written creating genesis 😢
Something must have messed up while committing